### PR TITLE
Add Infinite Scroll to our blog

### PR DESCRIPTION
**Pull request by SantosGuillamot**
Following the [Final Implementation](https://community.frontity.org/t/infinite-scroll-hooks/2055/32?u=santosguillamot) for the stable version of the Infinite Scroll hooks, I've implemented it in our blog. Apart from that, I've updated the dependencies and add the `redirections` setting.

### CODE REVIEW

**Comment by frontibotito** [comment]
Deploy preview for _website_ ready!

Built with commit 4b178e5b660327d4494abfa8221e07cfbaaf856b

https://frontity-org-qkrq3i7r1-frontity.vercel.app

**Comment by SantosGuillamot** [comment]
@luisherranz I assigned you because you are the supporter this week, but feel free to reassign it to @DAreRodz if you feel he should be the one reviewing it, as he worked on the hooks.

File: packages/twentytwenty-theme/src/components/archive/archive.js [hunk 1]
```diff
@@ -1,65 +1,29 @@
...
-        </>
-      )}
+      {pages.map(({ key, link, isLast, Wrapper }) => (
+        <Wrapper key={key}>
+          <ArchivePage link={link} />
+          {!isLast && <hr />}
+        </Wrapper>
+      ))}
+      {isFetching && <Loading />}
+      {isLimit && <button onClick={fetchNext}>Load Next Page</button>}
```
Comment (luisherranz) [inline packages/twentytwenty-theme/src/components/archive/archive.js:25]: If you don't use `limit` when you call `useArchiveInfiniteScroll()` you don't need this line 🙂

File: packages/twentytwenty-theme/src/components/post/post-list.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  return (
+    <div>
+      {posts.map(({ key, link, isLast, Wrapper }) => (
+        <Wrapper key={key}>
+          <Post link={link} />
+          {!isLast && <hr />}
+        </Wrapper>
+      ))}
+      {isFetching && <Loading />}
+      {isLimit && <button onClick={fetchNext}>Load Next Page</button>}
```
Comment (luisherranz) [inline packages/twentytwenty-theme/src/components/post/post-list.js:34]: Same here.

**Review by luisherranz — APPROVED** [review]
LGTM Mario!

File: packages/twentytwenty-theme/src/components/archive/archive.js (hunk 1 above)
Comment (SantosGuillamot) [reply packages/twentytwenty-theme/src/components/archive/archive.js]: You're totally right. I've just removed it :+1:

**Pull request merged**